yarn.lock

```diff
@@ -3400,14 +3400,6 @@ any-promise@^1.0.0:
   resolved "https://registry.yarnpkg.com/any-promise/-/any-promise-1.3.0.tgz#abc6afeedcea52e809cdc0376aed3ce39635d17f"
```

Comment:
could you remove the yarn.lock from the PR. I will go ahead and add it to the git ignore soon, just because it has given us some problems with merge conflicts in the past and will be generated when we do yarn install